R/epipredict-package.R

```diff
@@ -4,6 +4,10 @@
 #' @importFrom stats poly predict lm residuals quantile
 #' @importFrom cli cli_abort
 #' @importFrom lifecycle deprecated
+#' @importFrom checkmate assert assert_character assert_int assert_scalar
+#'             assert_logical assert_numeric assert_number assert_integer
+#'             assert_integerish assert_date assert_function
+#' @importFrom purrr walk
```

Comment:
We shouldn't need this (or `{purrr}` at all). See https://github.com/cmu-delphi/epipredict/blob/d46e0c5c42eaa2c76ae495ad709bcd62eca5e633/R/compat-purrr.R#L9